### PR TITLE
open-cluster-management/backlog#7400 dev input/change needed

### PR DESCRIPTION
**Pull request by swopebe**
--adding sample into the actual Ansible doc

### CODE REVIEW

File: manage_applications/ansible_config.adoc [hunk 1]
```diff
@@ -133,3 +131,33 @@ Four fields are defined in subscription.status.ansibleJobs:
 - prehookJobsHistory: All the prehook AnsibleJobs history
 - lastPosthookJobs: The most recent posthook AnsibleJobs
 - posthookJobsHistory: All the posthook AnsibleJobs history
+
+
+[#ansible-sample-yaml]
+== Ansibale sample YAML
```
Comment (mikeshng) [inline manage_applications/ansible_config.adoc]: spelling for `Ansibale`

File: manage_applications/ansible_config.adoc [hunk 2]
```diff
@@ -133,3 +131,33 @@ Four fields are defined in subscription.status.ansibleJobs:
...
 - lastPosthookJobs: The most recent posthook AnsibleJobs
 - posthookJobsHistory: All the posthook AnsibleJobs history
+
+
+[#ansible-sample-yaml]
+== Ansibale sample YAML 
+
+See the following sample:
+
+[source,yaml]
```
Comment (mikeshng) [inline manage_applications/ansible_config.adoc:141]: I think `,` should be `.`

File: manage_applications/ansible_config.adoc [hunk 3]
```diff
@@ -133,3 +131,33 @@ Four fields are defined in subscription.status.ansibleJobs:
...
+----
+apiVersion: tower.ansible.com/v1alpha1
+kind: AnsibleJob
+metadata:
+  generateName: demo-job- # generate a unique suffix per 'kubectl create'
```
Comment (mikeshng) [inline manage_applications/ansible_config.adoc]: this is my fault can you change it from
`generateName: demo-job- # generate a unique suffix per 'kubectl create'`
to
`name: demo-job-001`

**Review by mikeshng — APPROVED** [review]

File: manage_applications/ansible_config.adoc (hunk 1 above)
Comment (swopebe) [reply manage_applications/ansible_config.adoc]: puh!

File: manage_applications/ansible_config.adoc (hunk 2 above)
Comment (swopebe) [reply manage_applications/ansible_config.adoc:141]: not sure where you are seeing that, line #?

Comment (swopebe) [reply manage_applications/ansible_config.adoc:141]: In the code? I think that format was carried over from .md to adoc, but I will confirm.

Comment (mikeshng) [reply manage_applications/ansible_config.adoc:141]: line 141 + `[source,yaml]`

I think should be `[source.yaml]` but I am not sure please check. Thanks.

Comment (swopebe) [reply manage_applications/ansible_config.adoc:141]: Def across the doc I see us using the comma, and it is building. This is new to our code, as it seems to be an .adoc requirement but I have not investigated it other than it renders in review properly, and it builds. I will change if there is a reason to but seems comma is correct.

File: manage_applications/ansible_config.adoc (hunk 1 above)
Comment (swopebe) [reply manage_applications/ansible_config.adoc]: changed

File: manage_applications/ansible_config.adoc (hunk 3 above)
Comment (mikeshng) [reply manage_applications/ansible_config.adoc]: @bcpswope I forgot to clarify with the new name we don't need this comment anymore `# generate a unique suffix per 'kubectl create'` please remove this comment. Thanks.

Comment (swopebe) [reply manage_applications/ansible_config.adoc]: Ha, I had a comment that I didn't send asking you that exact question. Reading my mind now--worked together too long.

Comment (swopebe) [reply manage_applications/ansible_config.adoc]: merging, will send source for you to validate.

**Pull request merged**